### PR TITLE
use quadrille function chroma.limits to pass params to domain

### LINKED ISSUE #979: Bubble chart colors
**Is your feature request related to a problem? 
When there is not enough data the bubble chart has very similar colours
**Describe the solution you'd like**
we currently use(https://gka.github.io/chroma.js/#scale-domain)[ domain] property and the code looks like this:
```js
  colorScale: computed('model', function() {
    return chroma
      .scale(["deeppink", "lightblue", "limegreen"])
      .domain([this.get('minApprovalDelta'), 0, this.get('maxApprovalDelta') ]);
  }),
```
Im thinking to create average value function and convert it into something like this - this way the colours should be evenly spread on both sides of the average value - or maybe better to use: lower decile, median and upper decile - so if some data is waaaay of from the rest it won't influence the chart colours.
```js
  colorScale: computed('model', function() {
    return chroma
      .scale(["deeppink", "lightblue", "limegreen"])
      .domain([this.get('minApprovalDelta'), 
               this.get('avgApprovalDelta'),
               this.get('maxApprovalDelta')
              ]);
  }),
```
**Describe alternatives you've considered**
I was thinking about logarithmic but this might create similar problems when all the values start to grow
There is a helper function in chroma for that and it might use logharitmic or quartiles andcouple others [limits](https://gka.github.io/chroma.js/#chroma-limits)  - it all depends how the real data looks like because the seeded data is not useful right now

### PR DESCRIPTION
**Pull request by dkuku**
#### Description
changed way the array is calculated and passed to the bubble chart domain method
Fixes #979  - Without real data it's hard to test it
#### How Has This Been Tested?
I have no idea how to test it in ember - I'll probably can extract this into variable and create an unit test. Any suggestions welcome
#### Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by roschaefer** [comment]
This is how it looks like right now:
bubble-chart
Maybe we should add another issue to create an anonymized export of the database? Which we could share among the team?

**Comment by roschaefer** [comment]
I think replacing the `users` database table with fake data would be enough for an anonymized data set. Do you want to give it a try @dkuku? I could send you the export and you could improve the visualization.

**Comment by dkuku** [comment]
I used a [color scheme from http://colorbrewer2.org](http://colorbrewer2.org/#type=diverging&scheme=RdBu&n=5) Chroma supports all colour schemes from this website out of the box. 
I kinda like it - it's not too colorfull and warm colours mean that the broadcast is in the upper 50% cold colours in the lower. white are in the middle but you can see if it's over or under the median (there was a version with yellow in the middle but for me it was confusing having 3 colours. 
It's also colourblind friendly. 
2019-01-11-233329_1920x1080_scrot

File: frontend/app/controllers/visualize/time/index.js [hunk 1]
```diff
@@ -38,8 +39,11 @@ export default Controller.extend({
 
   colorScale: computed('model', function() {
     return chroma
-      .scale(["deeppink", "lightblue", "limegreen"])
-      .domain([this.get('minApprovalDelta'), 0, this.get('maxApprovalDelta')]);
+      .scale(chroma.scale('RdBu')
+        .colors(5)
+        .reverse()
```
Comment (roschaefer) [inline frontend/app/controllers/visualize/time/index.js]: My association with the colour red is danger, red traffic lights or bad. The first time I had a look on the chart I believed "red" means a decreased approval. How come you wanted to reverse the colour scale?

**Comment by roschaefer** [comment]
One more thing, maybe you want to update the preview image on https://rundfunk-mitbestimmen.de/visualize in meetings I've seen many people accidently clicking on the wrong chart. :wink:

**Comment by dkuku** [comment]
this is totally up to you-if the bubble would be heart shaped then red
makes more sense when you like something. But I could not imagine nothing
thats opposite to a heart. On the website I pasted red was on top as well
so I made it like that. You can always use asyncvoter and ask people what
they think in the channel.

**Comment by dkuku** [comment]
Will do that

File: frontend/app/controllers/visualize/time/index.js (hunk 1 above)
Comment (dkuku) [reply frontend/app/controllers/visualize/time/index.js]: @roschaefer Sorry but I had to ;) - I start to understand how d3 works - thanks
2019-01-13-143734_1920x1080_scrot

Comment (dkuku) [reply frontend/app/controllers/visualize/time/index.js]: @roschaefer What do you think to have a dropdown menu with colorschemes to choose from??
I just think where this should be implemented ? Globally on the visualisation path or per chart.
This way I can also have another dropdown to filter the data  - now we have tv and radio mixed up and maybe to choose the limits function

**Pull request merged**